### PR TITLE
Currency dropdown broken after replacing market address #1352

### PR DESCRIPTION
**Pull request by liaojikunwork**

### CODE REVIEW

**Review by pimato — APPROVED** [review]

**Review by Mi-Lan — APPROVED** [review]

File: app/src/hooks/useGraphMarketsFromQuestion.tsx [hunk 1]
```diff
@@ -50,13 +51,10 @@ export const useGraphMarketsFromQuestion = (questionId: string): Result => {
     if (!questionId) setMarkets([])
   }, [questionId])
 
-  if (data && data.question && data.question.indexedFixedProductMarketMakers.length > 0 && markets.length === 0) {
```
Comment (KadenZipfel) [inline app/src/hooks/useGraphMarketsFromQuestion.tsx:53]: Don't we want to leave this check: `data.question.indexedFixedProductMarketMakers.length > 0` so we can error if no markets are returned?

File: app/src/hooks/useGraphMarketsFromQuestion.tsx [hunk 2]
```diff
@@ -50,13 +51,10 @@ export const useGraphMarketsFromQuestion = (questionId: string): Result => {
...
+  if (data && data.question && !isObjectEqual(markets, data.question.indexedFixedProductMarketMakers)) {
     setMarkets(data.question.indexedFixedProductMarketMakers)
   } else if (data && data.question && !data.question.indexedFixedProductMarketMakers.length) {
-    return {
-      markets: [],
-      status: Status.Error,
-    }
+    setMarkets([])
```
Comment (KadenZipfel) [inline app/src/hooks/useGraphMarketsFromQuestion.tsx:57]: Shouldn't we still return an error here?

Comment (liaojikunwork) [reply app/src/hooks/useGraphMarketsFromQuestion.tsx:57]: @KadenZipfel , this section doesn't mean error.
It means "no markets" is returned.

File: app/src/hooks/useGraphMarketsFromQuestion.tsx (hunk 1 above)
Comment (liaojikunwork) [reply app/src/hooks/useGraphMarketsFromQuestion.tsx:53]: @KadenZipfel , if no markets are returned, indexedFixedProductMarketMakers will be an empty array and isObjectEqual will return true.
Don't need to check `data.question.indexedFixedProductMarketMakers.length > 0`

**Review by KadenZipfel — APPROVED** [review]

**Pull request merged**